docs/topics/spider-middleware.rst

```diff
@@ -328,6 +328,68 @@ Default: ``True``
 
 Whether to enable referer middleware.
 
+.. setting:: REFERER_POLICY
+
+REFERER_POLICY
+^^^^^^^^^^^^^^
+
+.. versionadded:: 1.4
+
+Default: ``'scrapy.spidermiddlewares.referer.DefaultReferrerPolicy'``
+
+.. reqmeta:: referrer_policy
```

Comment:
I guess the way to go is to rename the other option and a middleware itself as well; of course, it can be done later.